packs/twitter/actions/update_status.yaml

```diff
@@ -0,0 +1,11 @@
+---
+  name: "update_status"
+  runner_type: "run-python"
+  description: "Update your status (post a new tweet)."
+  enabled: true
+  entry_point: "update_status.py"
+  parameters:
+    status:
+      type: "string"
```

Comment:
Yep, Twitter supports unicode, but I believe jsonschema only has "string" type and Python json schema parser we use handles unicode just fine, right?